charts/ingress-nginx/templates/controller-ingressclass.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.controller.ingressClass.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  labels:
+    {{- include "ingress-nginx.labels" . | nindent 4 }}
+    app.kubernetes.io/component: controller
+    {{- with .Values.controller.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ include "ingress-nginx.controller.fullname" . }}
+{{- if .Values.controller.annotations }}
+  annotations: {{ toYaml .Values.controller.annotations | nindent 4 }}
+{{- end }}
+spec:
+  controller: kubernetes.io/ingress-controller
+  parameters:
+    apiGroup: k8s.example.com
+    kind: IngressParameters
+    name: external-lb
+{{- end -}}
```

Comment:
AFAIK params are not supported for nginx-ingress: https://github.com/kubernetes/ingress-nginx/issues/5593#issuecomment-721479598